legacy-sdk/whirlpool/src/errors/errors.ts

```diff
@@ -20,6 +20,8 @@ export enum SwapErrorCode {
   TickArrayCrossingAboveMax = `TickArrayCrossingAboveMax`,
   TickArrayIndexNotInitialized = `TickArrayIndexNotInitialized`,
   TickArraySequenceInvalid = `TickArraySequenceInvalid`,
+  AmountRemainingOverflow = `AmountRemainingOverflow`,
+  AmountCalcOverflow = `AmountCalcOverflow`,
```

Comment:
can we add the failure expectations to the `swap` method comments in `swap-ix.ts` & `ix.ts`

Reply:
good point!
added👍️ (these errors and recently added errors)